lib/shell.js

```diff
@@ -34,6 +34,8 @@ module.exports = function(cmd, args, options, callback) {
         }
       };
 
+  process.setMaxListeners(0);
```

Comment:
Do you see a problem with this? It was a way to get rid of a lot of annoying warnings about not being able to register more than 10 listeners to process calls.